include/chimera/util.h

```diff
@@ -315,6 +315,11 @@ std::pair<unsigned, unsigned> getFunctionArgumentRange(
  */
 bool hasNonPublicParam(const clang::CXXMethodDecl *decl);
 
+/**
+ * Converts a C++ built-in type name to the Python equivalent type name.
+ */
+std::string getPythonBuiltinTypeName(const std::string &type_name);
```

Comment:
Similarly, let us simply remove "Python" from this, as it is really just an arbitrary mapping as far as Chimera is concerned.

```suggestion
std::string getBuiltinTypeName(const std::string &type_name);
```